spec/unit/configurer_spec.rb

```diff
@@ -690,7 +690,8 @@ def expects_neither_new_or_cached_catalog
         @agent.instance_variable_set(:@environment, 'second_env')
         expects_new_catalog_only(@catalog)
 
-        Puppet.expects(:err).with(regexp_matches(/Could not/)).times(3)
+        @agent.stubs(:send_report)
+        @agent.stubs(:convert_catalog).returns(@catalog)
```

Comment:
Since this is a private method in configurer, can we not stub it, and instead stub the `to_ral` part?